python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -791,3 +812,51 @@ def info_string(self):
         lm_summary = self.load_metrics.summary()
         autoscaler_summary = self.summary()
         return "\n" + format_info_string(lm_summary, autoscaler_summary)
+
+
+class NodeTracker:
```

Comment:
Can we put this into a new file?

Reply:
+ 1 
autoscaler.py already looks monstrous